tests/IOST/SignerTests.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "HexCoding.h"
+#include "Base58.h"
+#include "proto/IOST.pb.h"
+#include "IOST/Signer.h"
+#include <gtest/gtest.h>
+
+#include <nlohmann/json.hpp>
+#include <iostream>
+
+namespace TW {
+namespace IOST {
+
+TEST(IOSTSigner, Sign) {
+    auto tx = Proto::Transaction();
+    tx.set_time(1550137587000000000);
+    tx.set_expiration(tx.time() + int64_t(1000000000) * 300);
+    tx.set_gas_ratio(1);
+    tx.set_gas_limit(1000000);
+    tx.set_chain_id(1024);
+
+    tx.add_actions();
+    auto action = tx.mutable_actions(0);
+    action->set_contract("token.iost");
+    action->set_action_name("transfer");
+
+    std::string token = "iost";
+    std::string src = "myname";
+    std::string dst = "admin";
+    std::string amount = "10";
+    std::string memo = "";
+
+    nlohmann::json j;
+    j.push_back(token);
+    j.push_back(src);
+    j.push_back(dst);
+    j.push_back(amount);
+    j.push_back(memo);
+    std::string data = j.dump();
+    ASSERT_EQ(data, std::string("[\"iost\",\"myname\",\"admin\",\"10\",\"\"]"));
```

Comment:
I have seen same json manipulation in kotlin/swift/cpp, could you please make it easier for coin transfer?

Reply:
done